src/templates/partials/kubelet.hbs

```diff
@@ -0,0 +1,8 @@
+# {{output.header}}
+# {{{output.link}}}
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+tlsCertFile: /path/to/ca.crt
+tlsPrivateKeyFile: /path/to/privkey.pem
+tlsCipherSuites: '{{{join output.ciphers ","}}}'
+tlsMinVersion: {{output.protocols.[0]}}
```

Comment:
The protocols need to be converted into Go format, see: https://github.com/mozilla/ssl-config-generator/pull/197#discussion_r1453940826

Reply:
```suggestion
tlsMinVersion: {{#if (eq output.protocols.[0] "TLSv1")}}VersionTLS10{{else}}{{{replace output.protocols.[0] "TLSv1." "VersionTLS1"}}}{{/if}}
```